scalableminds/webknossos
# use new sbt image

### Steps to test:
- compile & run tests in docker / check them in CI:
  - `docker-compose run --rm compile sbt clean compile stage`
  - `docker-compose run base yarn install --frozen-lockfile`
  - `docker-compose run base yarn test-verbose`
  - `docker-compose run --rm e2e-tests`
- check that caching works
  - for the sbt / maven / yarn caches
  - for the `webknossos-dev` image
    - locally
    - in the CI ([first build](https://circleci.com/gh/scalableminds/webknossos/4360) vs [build with cached dev image](https://circleci.com/gh/scalableminds/webknossos/4362)

### Open Questions:
* Should we actually cache the `webknossos-dev` image on dockerhub? It is negligibly faster than rebuilding completely, taking into account that it has to be pulled & pushed.

### Issues:
- relates to https://github.com/scalableminds/dockerfiles/issues/2 and https://github.com/scalableminds/dockerfiles/pull/9

------
